src/analytics/Properties.tsx

```diff
@@ -1290,9 +1290,7 @@ interface SwapEventsProperties {
     tokenSymbol?: string
   }
   [SwapEvents.swap_gas_fees_learn_more]: undefined
-  [SwapEvents.swap_screen_review_swap]: undefined
-  [SwapEvents.swap_review_screen_open]: SwapEvent & Web3LibraryProps & Partial<SwapTxsProperties>
-  [SwapEvents.swap_review_submit]: SwapQuoteEvent &
+  [SwapEvents.swap_screen_confirm_swap]: SwapQuoteEvent &
```

Comment:
maybe we can keep `swap_review_submit`? then we wouldn't need to update any data dashboards (since they do the same thing?)